pkg/config/consts.go

```diff
@@ -26,7 +26,7 @@ const (
 	MaxInvBlocks = 500
 
 	// Protocol-based consensus step time.
-	ConsensusTimeOut = 20 * time.Second
+	ConsensusTimeOut = 5 * time.Second
```

Comment:
back to 5s?

Reply:
Yes, this was the timeout prior to increase block time to 15secs